src/reducers/root-reducer.ts

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from "redux";
+import userReducer from "./user-reducer";
+import routeReducer from "./route-reducer";
+import waypointReducer from "./waypoint-reducer";
+
+export default combineReducers({
+	user: userReducer,
+	route: routeReducer,
+	waypoints: waypointReducer,
```

Comment:
Just minor style-nitpicks:
* the `root-reducer` file could be renamed to `index` so that we can do  
```typescript
import rootReducer from "…/reducers";
```
* the imported reducers here could just be named `user`, `route`, and `waypoints` — then we can do  
```typescript
import user from "./user-reducer";
…

export default combineReducers({ user, route, waypoints });
```

Reply:
Done